tests/database_metrics/test_parsers.py

```diff
@@ -0,0 +1,24 @@
+"""Tests for parsers.py"""
+import unittest
+
+# py3tester coverage target
+__test_target__ = 'delphi.operations.database_metrics.parsers'
+
+
+class TestParser(unittest.TestCase):
+
+    def test_parse_db_size(self):
+        test_query_result = b'db\tsize_mb\nepidata\t7.79687500\ninformation_schema\t0.18750000\n'
+        self.assertEqual(parse_db_size(test_query_result), 7.796875)
+
+    def test_metrics(self):
+        test_metrics = (b'db\tsize_mb\nepidata\t7.79687500\ninformation_schema\t0.18750000\n',
+                        1,
+                        [{"memory_stats": {"usage": 3}}, {"memory_stats": {"usage": 4}}])
+        print(parse_metrics(test_metrics))
```

Comment:
Probably want to remove this `print` and add an fstring as the failure message in the assert below

Reply:
Oops that wasn't supposed to be there, thanks for catching If it fails unittest should print out the missing dicts automatically in the assert.

Removed.